elastic/kibana
# [Uptime] [Synthetics Service] log errors when fetching service locations

## Summary

This PR logs errors when fetching locations from the Synthetics Service.

### Testing
1. Update manifest URL in `kibana.dev.yml`

```
xpack.uptime.service.manifestUrl: https://manifest.synthetics.gcp.qa.cld.elstc.co/v1/manifest.json
```

QA environment currently contains malformed JSON, so it'll trigger an error (the team has been notified though, so I'm not sure how much longer it'll stay broken.)

2. Observe the error in the Kibana logs
